### PR TITLE
add node-name flag to init in the case of aws pke

### PR DESCRIPTION
**Pull request by pbalogh-sa**
Signed-off-by: Peter Balogh <p.balogh.sa@gmail.com>

| Q               | A
| --------------- | ---
| Bug fix?        |yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| License         | Apache 2.0

### What's in this PR?
- add node-name flag in the case of aws pke

### CODE REVIEW

File: src/cluster/ec2_pke.go [hunk 1]
```diff
@@ -859,7 +859,8 @@ func (c *EC2ClusterPKE) GetBootstrapCommand(
 			"--kubernetes-api-server=%q "+
 			"--kubernetes-cluster-name=%q "+
 			"--kubernetes-master-mode=%q "+
-			"--kubernetes-advertise-address=0.0.0.0:6443",
+			"--kubernetes-advertise-address=0.0.0.0:6443 "+,
```
Comment (pregnor) [inline src/cluster/ec2_pke.go:862]: **Required**: The comma **MUST** be removed from here because it is syntactically incorrect.

**Review by pregnor — CHANGES_REQUESTED** [review]
Don't have much context knowledge, need to be manually tested.

File: src/cluster/ec2_pke.go [hunk 2]
```diff
@@ -859,7 +859,8 @@ func (c *EC2ClusterPKE) GetBootstrapCommand(
...
+			"--kubernetes-node-name=$(curl http://169.254.169.254/latest/meta-data/local-hostname) "
```
Comment (pregnor) [inline src/cluster/ec2_pke.go:863]: **Required**: A comma **MUST** be added here, because otherwise it is syntactically incorrect.

**Review by pregnor — CHANGES_REQUESTED** [review]

File: src/cluster/ec2_pke.go (hunk 1 above)
Comment (pbalogh-sa) [reply src/cluster/ec2_pke.go]: thx

**Review by pregnor — APPROVED** [review]
LGTM, but needs manual test

**Comment by akijakya** [comment]
> LGTM, but needs manual test

I'm doing that right now

**Comment by akijakya** [comment]
This change has been included in the PR that makes PKE 0.9.3 as default image.

**Pull request closed without merging**